### PR TITLE
Update documentation to SDK v1.0(244)

### PR DESCRIPTION
**Pull request by macmagician**
Merge iOS and tvOS into one document to reflect shared source.

### CODE REVIEW

**Review by fschurer — CHANGES_REQUESTED** [review]

**Review by rogerpales — APPROVED** [review]

File: docs/source/integrations/mobile/ios-fairplay-sdk.md [hunk 1]
```diff
@@ -182,17 +184,40 @@ The ContentKeyManager class should look like this:
...
+
+When using the same implementation as our demo application, iOS requests for licences for offfine content would reference the framework instance using the`ContentKeyManager`by calling:`ContentKeyManager.shared.assetResourceLoaderDelegate.doRequestPersistableContentKeys(contentKeyIDList: asset.stream.contentKeyIDList!, streamName: asset.stream.contentID, vudrmToken: asset.stream.vudrmToken!, avUrlAsset: avasset)`
 
 ##### DEMO
 
-With the VUDRMFairPlay framework installed, the example demo application uses entries in the `Streams.plist` file to configure the DRM. You can edit the `Streams.plist` file values, and/or add your own valid values, ensuring the `content_key_id_list` entry is correct, and that your `token` is valid (see **Preparation** above).
+The demo application that we use targets both iOS and tvOS platforms using shared source code. Each target platform references it's own version of the framework.
```
[the remark this answers is not in the document]
Comment (macmagician) [reply docs/source/integrations/mobile/ios-fairplay-sdk.md]: resolved commit a1fa0558f5a9887d2f38a9660840e7e7c0a36506

File: docs/source/integrations/mobile/ios-fairplay-sdk.md [hunk 2]
```diff
@@ -182,17 +184,40 @@ The ContentKeyManager class should look like this:
...
+
+With the VUDRMFairPlay framework installed, the example demo application reads entries in the `Streams.plist` file to configure the content and DRM. You can edit the `Streams.plist` file values, and/or add your own valid values, which should include a `content_id`, `playlist_url` and `vudrm_token`(see **Preparation** above).
+
+- The `content_key_id_list` entry in the `Streams.plist` is retrieved in the demo by parsing the manifest using the `getContentKeyIDList` function in the `AssetListTableViewController`.
+
+- The `name` value is the display name, which may differ from the `content_id`.
+
+- The `is_protected` boolean value indicates whether the content is protected with DRM or not. This value can be overrideen in source if all content is protected.
```
[the remark this answers is not in the document]
Comment (macmagician) [reply docs/source/integrations/mobile/ios-fairplay-sdk.md]: resolved commit a1fa0558f5a9887d2f38a9660840e7e7c0a36506

**Review by fschurer — APPROVED** [review]

File: docs/source/integrations/mobile/ios-fairplay-sdk.md [hunk 3]
```diff
@@ -182,17 +185,40 @@ The ContentKeyManager class should look like this:
...
 
 ##### DEMO
 
-With the VUDRMFairPlay framework installed, the example demo application uses entries in the `Streams.plist` file to configure the DRM. You can edit the `Streams.plist` file values, and/or add your own valid values, ensuring the `content_key_id_list` entry is correct, and that your `token` is valid (see **Preparation** above).
+The demo application that we use targets both iOS and tvOS platforms using shared source code. Each target platform references its own version of the framework.
+
+With the VUDRMFairPlay framework installed, the example demo application reads entries in the `Streams.plist` file to configure the content and DRM. You can edit the `Streams.plist` file values, and/or add your own valid values, which should include a `content_id`, `playlist_url` and `vudrm_token`(see **Preparation** above).
```
Comment (PerritonBranch) [inline docs/source/integrations/mobile/ios-fairplay-sdk.md:204]:
```suggestion
With the VUDRMFairPlay framework installed, the example demo application reads entries in the `Streams.plist` file to configure the content and DRM. You can edit the `Streams.plist` file values, and/or add your own valid values, which should include a `content_id`, `playlist_url`, and `vudrm_token`(see [Preparation](#preparation) above).
```

File: docs/source/integrations/mobile/ios-fairplay-sdk.md [hunk 4]
```diff
@@ -182,17 +185,40 @@ The ContentKeyManager class should look like this:
      
 ##### Online Playback / Download and Offline Playback:
 
-This will request a license for the content from the license server based on the type of token presented for each instance of VUDRMFairPlay. The tokens may be FairPlay Rental or FairPlay Persist. FairPlay Rental token license requests will be provided a streaming content key, which may be used for online streaming only. FairPlay Persist token license requests will provide a persistent content key, which may be used for both online streaming, and offline playback of downloaded content.
+The tvOS platform does not support downloading or offline playback.
+
+Requests for playback or download will request a license for the content from the license server based on the type of token presented for each instance of VUDRMFairPlay. The tokens may be FairPlay Rental or FairPlay Persist. FairPlay Rental token license requests will be provided a streaming content key, which may be used for online streaming only. FairPlay Persist token license requests will be provided a persistent content key, which may be used for both online streaming, and offline playback of downloaded content.
```
Comment (PerritonBranch) [inline docs/source/integrations/mobile/ios-fairplay-sdk.md:190]:
```suggestion
Requests for playback or download will result in a license request for the content from the license server, based on the type of token presented for each instance of VUDRMFairPlay. The tokens may be FairPlay Rental or FairPlay Persist. FairPlay Rental token license requests will be provided a streaming content key, which may be used for online streaming only. FairPlay Persist token license requests will be provided a persistent content key, which may be used for both online streaming, and offline playback of downloaded content.
```

**Review by PerritonBranch — CHANGES_REQUESTED** [review]

File: docs/source/integrations/mobile/ios-fairplay-sdk.md [hunk 5]
```diff
@@ -90,15 +92,16 @@ The demo project is based on Apple's FairPlay SDK example application. This is c
...
+ - An `skd://` URI entry in the `content_key_id_list`. The correct URI can be obtained by retrieving the manifest and parsing out the `URI` from the `EXT-X-SESSION-KEY` or `EXT-X-KEY`, this can be done in code (see our demo application for an example) or manually with curl in the Terminal app, e.g. enter: `curl https://eample.cloudfront.net/example-demo/examplecontent/examplecontent.ism/.m3u8`. 
  - The `playlist_url` to correctly prepared content.
- - A `name` for the content (the correct content ID will be parsed out by VUDRMFairPlay SDK and will always be the last path component of the above `skd://` URI entry).
- - A boolean value which indicates that the `Stream` object is protected with DRM. This setting would not be needed in scenarios where all content is protected, as all assets can be set as protected by default in the source code. 
+ - A `name` for the content. This will be the readable display name and is not used by the SDK. This may differ from the `contentID`.
```
[the remark this answers is not in the document]
Comment (macmagician) [reply docs/source/integrations/mobile/ios-fairplay-sdk.md]: resolved commit: facdd5bc708bdf51abc20cd1dda702e8531327c3

File: docs/source/integrations/mobile/ios-fairplay-sdk.md [hunk 6]
```diff
@@ -90,15 +92,16 @@ The demo project is based on Apple's FairPlay SDK example application. This is c
...
+ - A `content_id` for the content, being the same content ID that the content was prpeared with. This value will always be the last path component of the stream `content_key_id_list` or `skd://` URI entry. Our demo application shows how to retrieve and parse the `content_key_id_list`, however, the `contentID` always needs to be provided for use with offline assets.
```
[the remark this answers is not in the document]
Comment (macmagician) [reply docs/source/integrations/mobile/ios-fairplay-sdk.md]: resolved commit: facdd5bc708bdf51abc20cd1dda702e8531327c3

**Review by fschurer — DISMISSED** [review]

File: docs/source/integrations/mobile/ios-fairplay-sdk.md [hunk 7]
```diff
@@ -182,17 +185,40 @@ The ContentKeyManager class should look like this:
...
+The tvOS platform does not support downloading or offline playback.
+
+Requests for playback or download will result in a license request for the content from the license server, based on the type of token presented for each instance of VUDRMFairPlay. The tokens may be FairPlay Rental or FairPlay Persist. FairPlay Rental token license requests will be provided a streaming content key, which may be used for online streaming only. FairPlay Persist token license requests will be provided a persistent content key, which may be used for both online streaming, and offline playback of downloaded content.
+
+In iOS, when a download of an asset has been completed with a persist token policy, further playback requests will use the downloaded asset and associated content key, even when online.
 
-When a download of an asset has been completed with a persist token policy, further playback requests will use the downloaded asset and associated content key, even when online.
+There are significant limitations using AirPlay to stream any content to an Apple TV that has been downloaded to the users device. Please see the [Limitations]((#limitations)) section for further information.
```
Comment (PerritonBranch) [inline docs/source/integrations/mobile/ios-fairplay-sdk.md:194]:
```suggestion
There are significant limitations using AirPlay to stream any content to an Apple TV that has been downloaded to the users device. Please see the [Limitations](#limitations) section for further information.
```

**Review by PerritonBranch — CHANGES_REQUESTED** [review]

**Review by PerritonBranch — APPROVED** [review]

**Review by fschurer — APPROVED** [review]

**Pull request merged**